### PR TITLE
docs: modify fine-grain logger docs

### PR DESCRIPTION
**Pull request by botengyao**
Commit Message:
Additional Description:
Risk Level: low
Testing: 
Docs Changes: modify fine-grain logger docs
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: source/docs/fine_grain_log.md [hunk 1]
```diff
@@ -40,7 +40,25 @@ Runtime update of Fine-Grain Logger is supported with administration interface,
 3. `POST /logging?paths=file_path1:level1,file_path2:level2...`: Change log level of multiple file paths at once;
 4. `POST /logging?level=<level>`: Change levels of all loggers.
 
-Users can view and change the log level in a granularity of file in runtime through admin page. Note that `file_path` is determined by `__FILE__` macro, which is the path seen by preprocessor.
+Users can view and change the log level in a granularity of file in runtime through admin page.
```
Comment (KBaichoo) [inline source/docs/fine_grain_log.md:43]: s/in a granularity of file in runtime through admin page./by file granularity at runtime through the admin page./

File: source/docs/fine_grain_log.md [hunk 2]
```diff
@@ -40,7 +40,25 @@ Runtime update of Fine-Grain Logger is supported with administration interface,
...
+
+We also added the file basename, glob ``*`` and ``?`` support for fine-grain loggers. For example, we have the following active loggers with trace level.
```
Comment (KBaichoo) [inline source/docs/fine_grain_log.md:45]: s/We also added the/Envoy admin page supports the/

Comment (KBaichoo) [reply source/docs/fine_grain_log.md:45]: Can you also clarify the other "we"

**Review by KBaichoo — COMMENTED** [review]
Thanks mostly looks good, just some minor clarification.

File: source/docs/fine_grain_log.md (hunk 1 above)
Comment (botengyao) [reply source/docs/fine_grain_log.md]: Done.

File: source/docs/fine_grain_log.md (hunk 2 above)
Comment (botengyao) [reply source/docs/fine_grain_log.md]: Done

**Review by KBaichoo — APPROVED** [review]
Thank you!

**Pull request merged**